### ISSUE TITLE
Add support vor Webdriverio v5

### ISSUE THREAD

**Issue by cbou** [webdriverio/cucumber-boilerplate#250]
# Issue description
> Webdriverio v5 is release, it would be nice to have this repository updated to this version

https://webdriver.io/blog/2018/12/19/webdriverio-v5-released.html

**Comment by christian-bromann** [comment]
This is blocked on https://github.com/webdriverio/webdriverio/issues/2872

**Comment by abjerstedt** [comment]
I've done a bunch of work to port the boilerplate for v5 (this was an exercise to test my v5 implementation).

It is currently pending the v5 launch, and a few MINOR changes.

**Comment by xyngfei** [comment]
@abjerstedt, thx for your help to contribute to this library. 
May I know what is the current status? I really keen to use it for v5 implementation.

**Comment by abjerstedt** [comment]
@xyngfei put up a pr to push unit tests to my main pr. I've done the majority of the work... i just dont have time to push it over the finish line.

**Comment by xyngfei** [comment]
@abjerstedt, let me try it out first. 
Do you have any suggestion for any newbie how to get start to contribute to this repo?
